client/js/components/SelfGuidedItem/SelfGuidedItem.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {
+  TouchableOpacity,
+  Text,
+  View,
+  Image,
+  ImageBackground,
+} from 'react-native';
+
+import styles from './styles';
+
+const SelfGuidedItem = ({selfGuidedItem, navigation}) => (
+  <TouchableOpacity
+    onPress={() => {
+      if (selfGuidedItem) {
+        return navigation.push('SingleSelfGuided', {
+          itemId: selfGuidedItem.id,
+        });
+      }
+    }}
+    style={styles.container}>
+    <ImageBackground
+      source={{uri: selfGuidedItem.image}}
+      style={{
+        width: '100%',
+        height: '100%',
```

Comment:
Please move your inline styles to styles.js

Reply:
Fixed